src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Represents the file to store tasks the user has inputted.
+ */
+public class Storage {
+    private static final String DEFAULT_FILEPATH = "./data/duke.txt";
+
+    /**
+     * Creates a storage file at the intended file path.
+     */
+    public Storage() {
```

Comment:
Is this redundant?

Reply:
Or can add an extra constructor for when user wants to allocate multiple chiwas with each having their own storage.